src/services/doppler-sites-client.ts

```diff
@@ -0,0 +1,63 @@
+import { AxiosInstance, AxiosStatic } from 'axios';
+import { ResultWithoutExpectedErrors } from '../doppler-types';
+
+export interface DopplerSitesClient {
+  getBannerData(lang: string, type: string, page?: string | null): Promise<PromotionsResult>;
+}
+
+export interface Promotions {
+  title: string;
+  functionality: string;
+  description: string;
+  imageUrl: string;
+  backgroundUrl: string;
+  fontColor: string;
+}
+
+export type PromotionsResult = ResultWithoutExpectedErrors<Promotions>;
+
+export class HttpDopplerSitesClient implements DopplerSitesClient {
+  private readonly axios: AxiosInstance;
+  private readonly baseUrl: string;
+
+  constructor({ axiosStatic, baseUrl }: { axiosStatic: AxiosStatic; baseUrl: string }) {
+    this.baseUrl = baseUrl;
+    this.axios = axiosStatic.create({
+      baseURL: baseUrl,
+      withCredentials: true,
```

Comment:
I think that we do not need credentials here.

Reply:
update